modyn/supervisor/internal/triggers/performance/decision_policy.py

```diff
@@ -97,13 +97,21 @@ def evaluate_decision(
         mode: TriggerEvaluationMode,
         method: ForecastingMethod,
     ) -> bool:
-        threshold = performance_tracker.forecast_expected_performance(mode) - self.config.allowed_deviation
+        expected = performance_tracker.forecast_expected_performance(mode)
```

Comment:
as discussed, this call is invalid

Reply:
what i dont understand still is why in the hindsight mode we need any forecasting (let it be the rolling average). i understood it as "if our accuracy on the evaluation interval deviates too much, we trigger". but apparently something more is going on?

Reply:
With 
```python
forecast_expected_performance(
            mode, method="rolling_average"
        )
```
this is equivalent to saying, if current values deviates more than X % of the rolling average, then trigger. We simply just reuse the averaging from the forecasting method. It's not a true forecast.

Reply:
as discussde, let's rename expected to "optimal" performance for this